Libraries/ART/ARTNode.m

```diff
@@ -14,7 +14,16 @@ @implementation ARTNode
 - (void)insertReactSubview:(UIView *)subview atIndex:(NSInteger)atIndex
```

Comment:
Shouldn't this be RCTUIView* ?  Is your branch not synced with Mark's PR https://github.com/microsoft/react-native/pull/127/files that completed?

Reply:
Correct. Mark's wasn't in when I began this so it wasn't synced.

Reply:
Whoops commented in the wrong spot but no it's not synced with Marks. When I sync, this will update